cranelift/filetests/src/zkasm_runner.rs

```diff
@@ -73,10 +73,21 @@ pub fn run_zkasm_path(input_path: &Path) -> anyhow::Result<Vec<ExecutionResult>>
         include_str!("../../zkasm_data/generated/helpers/2-exp.zkasm"),
     )?;
 
+    // The node module necessary to execute zkAsm lives in `wasmtime/tests/zkasm/package.json`.
+    // We are trying to create a path to it that would work regardless of the current working
+    // directory that the caller is using.
+    let node_module_path = Path::new(file!())
+        .parent()
+        .unwrap()
+        .join("../../../../../tests/zkasm/")
+        .to_str()
+        .unwrap()
+        .to_string();
```

Comment:
```suggestion
        .display()
        .to_string();
```

Ideally this did not force filesystem paths to be utf-8, but I can imagine that `npm` or `node` in general won't take kindly to it anyway, so whatever. And I don't think any of us will have non-utf-8 working directories...

That said there's a caveat with `file!` -- it is affected by `remap-path-prefix` so if for any reason that flag is used, this code will most likely break. An alternative could be to use `env!("CARGO_MANIFEST_DIR")` and base the path from that.

Reply:
Both great points, thanks! I ended up switching to `CARGO_MANIFEST_DIR` which uses an absolute path.